src/main/java/core/basesyntax/CountFigure.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class CountFigure {
+    public static final int CountFigure = 5;
+    private ColorProducer colorProducer = new ColorProducer();
+
+    public Figure get() {
+        int randomFigureIndex = new Random().nextInt(CountFigure);
+        switch (randomFigureIndex) {
+            case 0:
+                return new Square(random(), colorProducer.getColor());
+            case 1:
+                return new RightTriangle(random(), random(), colorProducer.getColor());
+            case 2:
+                return new Rectangle(random(), random(), colorProducer.getColor());
+            case 3:
+                return new Circle(random(), colorProducer.getColor());
+            default:
+                return new IsoscelesTrapezoid(random(),
+                        random(), random(), colorProducer.getColor());
+        }
+    }
+
+    public int random() {
```

Comment:
we usually use verbs for method names. And if we use this method only in this class it's better to declare it as a private